doc/source/PreConfiguredSites.rst

```diff
@@ -98,53 +98,10 @@ The following is required for building new spack environments with any supported
 
    # To access /apps/contrib/spack-stack directory, first login to orion-devel-1 or orion-devel-2 login node.
    # Then sudo to role-epic account.
-   module purge
-
-``spack-stack`` module files on **orion** require a one-time modification before they will properly load. These module files rely on a system-provided module file that alters the environment variable ``MODULEPATH`` in such a way that it prevents the expected loading of ``spack-stack`` modules. This is only necessary for *Intel oneAPI* environment module files. After the stack is created, modify the stack modules as follows:
 
-.. code-block:: console
+   module purge
+   module load spack-managed-x86-64_v3/v1.0
```

Comment:
Does the wiki also need to be updated (i.e. do we need to load spack-managed-x86-64_v3/v1.0 before the spack-stack module use & module load commands)?